modules/ROOT/pages/visualizer-app-network.adoc

```diff
@@ -1,11 +1,9 @@
-= Visualizer Application Network
+= Use-case Specific Visualizations
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Use Anypoint Visualizer to view a graph of your application network.
-
-Anypoint Visualizer automatically generates the topology of the application.
+Use Anypoint Visualizer to view a graph of your application network. The _base unit_ is a Mule application that has been deployed and is running on a supported Mule runtime.
 
 To maximize visualization readability, consumer nodes (nodes that have only outbound connections) are shown at the top. Backend systems (nodes that have only inbound connections) are shown at the bottom.
```

Comment:
The spacing looked a bit weird to me here. Hard to tell in this mode though. Perhaps you can double check?

Reply:
Hm, it looks okay to me. I will check the build.